Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -14,6 +14,26 @@
  * limitations under the License.
  */
 
+/* NB: proto bytes were created via:
+     echo 'TEXT_FORMAT_PROTO' \
+       | ./build/external/protobuf/src/protobuf-build/src/protoc \
+           -I./Firestore/Protos/protos \
+           -I./build/external/protobuf/src/protobuf/src/ \
+           --encode=google.firestore.v1beta1.Value \
+           google/firestore/v1beta1/document.proto \
+           > output.bin
```

Comment:
I'd suggest the final step here would be `| hexdump` or `| od -t xC` or something like that.

Reply:
Done ( `| hexdump -C` )